diff --git a/src/exec.c b/src/exec.c
index c0a0b4a..3c8cce2 100644
--- a/src/exec.c
+++ b/src/exec.c
@@ -742,13 +742,17 @@ h_cmd (ed)
       return ERR;
     }
   COMMAND_SUFFIX (ed->display->io_f, ed);
-  if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
-    {
+  /*
+   * if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
+   *   {
+   */
       if (ed->exec->err)
         puts (ed->exec->err);
-    }
-  else
-    puts (ed->exec->err ? ed->exec->err : _("No previous error"));
+  /*
+   *   }
+   * else
+   *   puts (ed->exec->err ? ed->exec->err : _("No previous error"));
+   */
   return ed->display->io_f;
 }
 
@@ -838,10 +842,12 @@ l_cmd (ed)
   else if ((status = is_valid_range (ed->state->dot, ed->state->dot, ed)) < 0)
     return status;
   COMMAND_SUFFIX (ed->display->io_f, ed);
-  if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
+  /* if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL)) */
     ed->display->io_f = LIST;
-  else
-    ed->display->io_f |= LIST;
+  /*
+   * else
+   *   ed->display->io_f |= LIST;
+   */
   return display_lines (ed->exec->region->start,
                         ed->exec->region->end, ed);
 }
@@ -926,10 +932,12 @@ n_cmd (ed)
            < 0)
     return status;
   COMMAND_SUFFIX (ed->display->io_f, ed);
-  if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
+  /* if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL)) */
     ed->display->io_f = NMBR;
-  else
-    ed->display->io_f |= NMBR;
+  /*
+   * else
+   *   ed->display->io_f |= NMBR;
+   */
   return display_lines (ed->exec->region->start,
                         ed->exec->region->end, ed);
 }
@@ -968,10 +976,12 @@ p_cmd (ed)
   else if ((status = is_valid_range (ed->state->dot, ed->state->dot, ed)) < 0)
     return status;
   COMMAND_SUFFIX (ed->display->io_f, ed);
-  if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
+  /* if (ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL)) */
     ed->display->io_f = PRNT;
-  else
-    ed->display->io_f |= PRNT;
+  /*
+   * else
+   *   ed->display->io_f |= PRNT;
+   */
   return display_lines (ed->exec->region->start,
                         ed->exec->region->end, ed);
 }
@@ -1387,9 +1397,11 @@ w_cmd (ed)
            * Per SUSv4, exit_status cannot be reset. We'll do it
            * anyhow ...
            */
-          if (!(ed->exec->opt &
-                (POSIXLY_CORRECT|TRADITIONAL|SCRIPTED|EXIT_ON_ERROR)))
-            ed->exec->status = 0;
+          /*
+           * if (!(ed->exec->opt &
+           *       (POSIXLY_CORRECT|TRADITIONAL|SCRIPTED|EXIT_ON_ERROR)))
+           *   ed->exec->status = 0;
+           */
         }
       if (ed->file->is_glob || cy == 'n' || cy == 'p')
         printf (ed->exec->opt & SCRIPTED ? "" : "%s\n",
diff --git a/src/sub.c b/src/sub.c
index 3ad60ac..09cd896 100644
--- a/src/sub.c
+++ b/src/sub.c
@@ -38,8 +38,10 @@ resubstitute (s_nth, s_mod, s_f, sgpr_f, ed)
   int g_f = 0;                  /* Set if global toggled - e.g. `sg' */
 
   /* Return unless ed->input contains short-form substitution. */
+  /*
   if ((ed->exec->opt & (POSIXLY_CORRECT | TRADITIONAL))
       || !strchr (SGPR_CHARS, (unsigned) *ed->input))
+  */
     return 0;
 
   *sgpr_f = SGPR;               /* Flag short-form substitution. */
