--- Makefile.am.orig	2025-01-20 07:16:23.264463456 -0500
+++ Makefile.am	2025-01-20 07:17:00.344479132 -0500
@@ -744,11 +744,23 @@
 
 ## begin gnulib module regex
 
-if GL_COND_OBJ_REGEX
+# if GL_COND_OBJ_REGEX
+if LIBADD_REGEX
 libgnu_a_SOURCES += regex.c
+BUILT_SOURCES += regex.h dynarray.h malloc/dynarray.h
+DISTCLEANFILES += regex.h dynarray.h malloc/dynarray.h
+AM_CPPFLAGS += -DRE_ENABLE_I18N=1
+regex.h: regex.h.in
+	$(LN_S) -f $@.in $@
+
+dynarray.h: dynarray.h.in
+	$(LN_S) -f $@.in $@
+
+malloc/dynarray.h: malloc/dynarray.h.in
+	$(LN_S) -f dynarray.h.in malloc/dynarray.h
 endif
 
-EXTRA_DIST += regcomp.c regex.h regex_internal.c regex_internal.h regexec.c
+EXTRA_DIST += dynarray.h.in malloc/dynarray.h.in regcomp.c regex.h.in regex_internal.c regex_internal.h regexec.c
 
 EXTRA_libgnu_a_SOURCES += regcomp.c regex_internal.c regexec.c
 
@@ -1762,7 +1762,11 @@
 	  fi; \
 	done; \
 	:
-distclean-local: distclean-gnulib-libobjs
-distclean-gnulib-libobjs:
-	-rm -f @gl_LIBOBJDEPS@
-maintainer-clean-local: distclean-gnulib-libobjs
+
+clean-local:
+	rm -f @gl_LIBOBJDEPS@
+
+distclean-local: clean-local
+	-rm -f Makefile
+
+maintainer-clean-local: distclean-local
