--- Makefile.am.new	2018-07-24 03:01:33.000000000 -0400
+++ Makefile.am	2018-07-24 03:51:36.000000000 -0400
@@ -450,7 +442,22 @@
 
 ## begin gnulib module regex
 
+if LIBADD_REGEX
+BUILT_SOURCES += regex.h dynarray.h malloc/dynarray.h
+DISTCLEANFILES += regex.h dynarray.h malloc/dynarray.h
+AM_CPPFLAGS += -DRE_ENABLE_I18N=1
+regex.h: regex.h.in
+	$(LN_S) -f $@.in $@
+
+dynarray.h: dynarray.h.in
+	$(LN_S) -f $@.in $@
+
+malloc/dynarray.h: malloc/dynarray.h.in
+	$(LN_S) -f dynarray.h.in malloc/dynarray.h
+
-EXTRA_DIST += regcomp.c regex.c regex.h regex_internal.c regex_internal.h regexec.c
+endif LIBADD_REGEX
+
+EXTRA_DIST += dynarray.h.in malloc/dynarray.h.in regcomp.c regex.c regex.h.in regex_internal.c regex_internal.h regexec.c
 
 EXTRA_libgnu_a_SOURCES += regcomp.c regex.c regex_internal.c regexec.c
 
